pom.xml

```diff
@@ -28,9 +28,10 @@
     <version>40</version>
   </parent>
 
+  <!-- updated ztesler, 04-2016 -->
   <groupId>org.apereo.application</groupId>
   <artifactId>ImageQuiz</artifactId>
-  <version>1.0-SNAPSHOT</version>
+  <version>2.0</version>
```

Comment:
Since there's been no 1.0.0 release, I think the `<version>` shouldn't increment at all. Start observing Semantic Versioning after `1.0.0`. Before then it's just a wild west snapshot.